### PR TITLE
[production/RRFS.v1] physics updates for RRFS.v1 code freeze

### PR DESCRIPTION
**Pull request by haiqinli**
This PR includes the following updates:
1). MYNN updates;
2). RUC LSM updates (https://github.com/ufs-community/ccpp-physics/pull/163);
3). GF updates for humidity DA; restore scale-awareness in the 1st hour; suppress weak radar reflectivity over water;
4). smoke plume rise updates for stability on WCOSS2;
5). GWD updates to reduce the high 10m wind speed bias.

### CODE REVIEW

**Comment by MatthewPyle-NOAA** [comment]
@grantfirl Does more need to be done before this PR will be reviewed/merged?  Still trying to understand the process with the production release branch.  Thanks!

**Comment by grantfirl** [comment]
@MatthewPyle-NOAA We just need to review it and @haiqinli needs to upload the test_changes.list file in https://github.com/ufs-community/ufs-weather-model/pull/2147, assuming that we don't request changes. I'm also working on getting https://github.com/ufs-community/ccpp-physics/pull/147 and https://github.com/ufs-community/ccpp-physics/pull/159 in

**Comment by MatthewPyle-NOAA** [comment]
Thanks for the explanation, @grantfirl

File: physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.F90
Comment (grantfirl) [inline physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.F90]: These are only whitespace changes. Is that intended? If not, please revert this change.

File: physics/smoke_dust/rrfs_smoke_wrapper.F90
Comment (grantfirl) [inline physics/smoke_dust/rrfs_smoke_wrapper.F90]: Why are you adding `totprcp` when you're not using it?

File: physics/smoke_dust/rrfs_smoke_wrapper.meta
Comment (grantfirl) [inline physics/smoke_dust/rrfs_smoke_wrapper.meta]: Same question. Please remove this change if you're not using `totprcp`.

**Comment by grantfirl** [comment]
@haiqinli Looks fine with the exception of some minor typo-like things. Once you address those, I'll approve. Please remember to commit the test_changes.list file that is generated now in the ufs-weather-model/tests directory for that PR.

File: physics/smoke_dust/rrfs_smoke_wrapper.meta
Comment (haiqinli) [reply physics/smoke_dust/rrfs_smoke_wrapper.meta]: @grantfirl I will remove this unused variable. Thanks.

File: physics/smoke_dust/rrfs_smoke_wrapper.F90
Comment (haiqinli) [reply physics/smoke_dust/rrfs_smoke_wrapper.F90]: @grantfirl I will remove this unused variable. Thanks.

File: physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.F90
Comment (haiqinli) [reply physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.F90]: @grantfirl I will revert this. Thanks.

**Review by grantfirl — APPROVED** [review]

**Comment by grantfirl** [comment]
This contains the changes from https://github.com/ufs-community/ccpp-physics/pull/163.

File: physics/GWD/drag_suite.F90
Comment (yangfanglin) [inline physics/GWD/drag_suite.F90:1367]: @haiqinli @mdtoyNOAA Will this change affect global and HAFS applications as well ?  Have any tests been done to check the impact on applicaitons other than RRFS ?

Comment (mdtoyNOAA) [reply physics/GWD/drag_suite.F90:1367]: We tested this change in the HR3 C768 FV3GFS and detected improvements to the 500hPa ACC scores.

Comment (yangfanglin) [reply physics/GWD/drag_suite.F90:1367]: Mike, that is good know.  I'd assume this has major impact on surface winds and turbulent exchange coefficients as well.  Did you also see positive impact on grid-to-obs stats ?

Comment (yangfanglin) [reply physics/GWD/drag_suite.F90:1367]: Mike, since this change is hardwired and will be include in HR4.  Please record this change in our tracking spreadsheet.

Comment (MatthewPyle-NOAA) [reply physics/GWD/drag_suite.F90:1367]: @yangfanglin Keep in mind that this PR is just targeting the RRFS.v1 branch.

Comment (yangfanglin) [reply physics/GWD/drag_suite.F90:1367]: @MatthewPyle-NOAA I see. So the change is made locally and will not be committed back to the main repo.

Comment (MatthewPyle-NOAA) [reply physics/GWD/drag_suite.F90:1367]: @yangfanglin It may be included in a future PR targeting the main repo, but that will be a separate, future step.

Comment (mdtoyNOAA) [reply physics/GWD/drag_suite.F90:1367]: > Mike, that is good know. I'd assume this has major impact on surface winds and turbulent exchange coefficients as well. Did you also see positive impact on grid-to-obs stats ?

In the 10m winds we see significant improvements in RMSE in both west and east CONUS.  However, a slight degradation of windspeed bias in west (little change in bias in east).

Comment (mdtoyNOAA) [reply physics/GWD/drag_suite.F90:1367]: For VSDB verification, go to:
https://www.emc.ncep.noaa.gov/gmb/jhan/vsdbw/hr3gwd01/
The modifications to TOFD are the experiment "HR3GWD01".  "HR2D13" is HR2.

**Comment by jkbk2004** [comment]
tests are done at https://github.com/ufs-community/ufs-weather-model/pull/2147. @grantfirl can you merge this pr?

**Comment by jkbk2004** [comment]
@MatthewPyle-NOAA You may go ahead to merge this pr if you have permission.

**Comment by MatthewPyle-NOAA** [comment]
@jkbk2004 I don't have that permission, at least not yet.

**Comment by climbfuji** [comment]
I'll merge it for you

**Comment by jkbk2004** [comment]
> I'll merge it for you

Thanks! @climbfuji

**Comment by grantfirl** [comment]
@climbfuji Thanks. I got a late start this morning.

**Pull request merged**
